docs/sql-ref-syntax-aux-refresh-function.md

```diff
@@ -0,0 +1,60 @@
+---
+layout: global
+title: REFRESH FUNCTION
+displayTitle: REFRESH FUNCTION
+license: |
+  Licensed to the Apache Software Foundation (ASF) under one or more
+  contributor license agreements.  See the NOTICE file distributed with
+  this work for additional information regarding copyright ownership.
+  The ASF licenses this file to You under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with
+  the License.  You may obtain a copy of the License at
+ 
+     http://www.apache.org/licenses/LICENSE-2.0
+ 
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+---
+
+### Description
+
+`REFRESH FUNCTION` statement invalidates the cached function entry, which include class name
+and resource location of the given function. The invalidated cache is populated right away.
+Note that, refresh function only works for permanent function. Refresh native function or temporary function will cause exception.
```

Comment:
Sorry, the suggestion I gave you yesterday has a few grammar mistakes.

which include class name -> which includes the class name

Note that, refresh function only works for permanent function. -> Note that `REFRESH FUNCTION` only works for permanent functions.

Refresh native function or temporary function will cause exception. -> 
Refreshing native functions or temporary functions will cause an exception.